spec/features/users/regenerate_personal_key_spec.rb

```diff
@@ -119,10 +118,8 @@
       click_acknowledge_personal_key
 
       expect_confirmation_modal_to_appear_with_first_code_field_in_focus
-
-      press_tab
-
-      expect_continue_button_to_be_in_focus
+      expected_button_order = %w[Continue Back]
+      expect(all(:button).map(&:text).reject(&:empty?)).to eq expected_button_order
```

Comment:
For some reason, after pressing tab, Chrome didn't see that the continue button was in focus. What we want to test here is that on a mobile device, the continue button appears first, and on a desktop screen, it appears after the Back button. Iterating through all buttons will find them in order, so we can do that instead trying to tab through the elements.